app/battle/start.py

```diff
@@ -0,0 +1,16 @@
+from app.knight.creation import Knight
+
+
+def check_hp(knights: list[Knight]) -> list[Knight]:

```

Comment:
The function `check_hp` is correctly defined with type annotations. Ensure that the list of knights is passed correctly when calling this function.